spec/models/activity_spec.rb

```diff
@@ -1,6 +1,13 @@
 require "rails_helper"
 
 RSpec.describe Activity, type: :model do
+  describe "#finance" do
```

Comment:
This spec is added here and I think is better placed along with the implementation?

Reply:
Yes, this one slipped in here by accident. I'll put it in the right commit. Thanks

Reply:
This test now lives in the right commit. Thank you @mec